mailer.js

```diff
@@ -237,10 +244,70 @@ module.exports = function (log) {
       headers: {
         'X-Link': link
       },
-      subject: gettext('A new device is now syncing to your Firefox Account'),
+      subject: gettext('New sign-in to Sync'),
       template: 'newSyncDeviceEmail',
       templateValues: {
-        resetLink: link
+        resetLink: link,
+        supportUrl: this.supportUrl
+      },
+      uid: message.uid
+    })
+  }
+
+  Mailer.prototype.postVerifyEmail = function (message) {
+    log.trace({ op: 'mailer.postVerifyEmail', email: message.email, uid: message.uid })
+
+    var link = this.syncUrl
+
+    return this.send({
+      acceptLanguage: message.acceptLanguage,
+      email: message.email,
+      headers: {
+        'X-Link': link
+      },
+      subject: gettext('Firefox Account Verified'),
+      template: 'postVerifyEmail',
+      templateValues: {
+        link: link,
+        androidUrl: this.androidUrl,
+        iosUrl: this.iosUrl,
+        supportUrl: this.supportUrl
+      },
+      uid: message.uid
+    })
+  }
+
+  Mailer.prototype.reminderEmail = function (message) {
+    log.trace({ op: 'mailer.reminderEmail', email: message.email, uid: message.uid })
```

Comment:
Part of me says, when we send more than one kind I'll care, the other part capitulates to your infernally sound logic.